src/pages/Profile/Modal/SubmitModal.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect, useContext } from 'react';
+import { ActionText, AttentionTextModal, ColumnContainer, InputsTitleModal, RowContainer, TitleModal } from '../../../globalStyles';
+import { useHttp } from '../../../hooks/http.hook';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/OrangeMGE-ITP/SmartQ-ITP-Front/pullRequest?prid=8748018)
```suggestion
import { useHttp } from "../../../hooks/http.hook";
```